cdap-docs/introduction/source/index.rst

```diff
@@ -0,0 +1,1131 @@
+        |                     |                     |                     | entStream.converted |                     |
+        |                     |                     |                     | ","explore.output.f |                     |
+        |                     |                     |                     | ormat":"org.apache. |                     |
+        |                     |                     |                     | hadoop.hive.ql.io.a |                     |
+        |                     |                     |                     | vro.AvroContainerOu |                     |
+        |                     |                     |                     | tputFormat","output |                     |
+        |                     |                     |                     | .format":"org.apach |                     |
+        |                     |                     |                     | e.avro.mapreduce.Av |                     |
+        |                     |                     |                     | roKeyOutputFormat", |                     |
+        |                     |                     |                     | "explore.input.form |                     |
+        |                     |                     |                     | at":"org.apache.had |                     |
+        |                     |                     |                     | oop.hive.ql.io.avro |                     |
+        |                     |                     |                     | .AvroContainerInput |                     |
+        |                     |                     |                     | Format","explore.en |                     |
+        |                     |                     |                     | abled":"true"}}]    |                     |
+        +=============================================================================================================+
+        
+       |non-breaking-space|
+
+
+.. list-table::
+   :widths: 15 65 20
+
+   * - **Current Approach**
+     - - Write a custom consumer that reads from source (Example: Kafka)
+       - Write the data to HDFS
+       - Create external table in Hive called ``stream_ip2geo``
+     - - HDFS
+       - Hive
+       - Kafka
+
+.. list-table::
+   :widths: 15 85
+
+   * - **CDAP**
+     - ``$ load stream logEventStream examples/resources/accesslog.txt``
+       ::
+
+        Successfully sent stream event to stream 'logEventStream'
+
+       |non-breaking-space|
+
+.. list-table::
+   :widths: 15 65 20
+
+   * - **Current Approach**
+     - - Run commands using HBase shell:
+       - ``hbase shell> list``
+       - ``hbase shell> hdfs fs -ls /path/to/my/files``
+     - - HBase
+       - HDFS
+
+.. list-table::
+   :widths: 15 85
+
+   * - **CDAP**
+     - Dataset that is time partitioned::
+     
+        $ list dataset instances
```

Comment:
This got rendered funny in the resulting html: `$ list dataset instances - Dataset that is time paritioned`